protocols/relay/src/priv_client.rs

```diff
@@ -46,7 +46,9 @@ use std::ops::DerefMut;
 use std::pin::Pin;
 use std::task::{Context, Poll};
 
-/// The events produced by the [`Client`] behaviour.
+pub use transport::Transport;
```

Comment:
We can remove the `pub` here in favor of accessing the `Transport` directly in `lib.rs` for the re-export.